msm/msm

```diff
@@ -176,56 +260,74 @@ function install() {
       fi
     fi
     if [[ -f "requirements.txt" ]]; then
-      echo "Installing requirements..."
-      if [[ "${picroft}" == 'false' ]]; then
+      echo "  Installing requirements..."
+      if [[ "${picroft}" == "false" ]]; then
```

Comment:
Something is weird here. On my desktop system the script entered the else branch of this if-statement and installed requirements using `sudo`

Debug output:
```
+ echo '  Installing requirements...'
  Installing requirements...
+ [[ '' == \f\a\l\s\e ]]
+ sudo pip install -r requirements.txt
```

Reply:
I guess it's that the `picroft` variable has been renamed `picroft_mk1`

Reply:
Fixed